src/test_l1_dsVideoPort.c

```diff
@@ -1101,125 +1101,6 @@ void test_l1_dsVideoPort_negative_dsIsVideoPortActive(void) {
 	UT_LOG("\n Out %s\n", __FUNCTION__); 
 }
 
-
-/**
- * @brief Positive Test Scenarios for dsEnableDTCP()
- * 
- * **Test Group ID:** Basic: 01@n
- * **Test Case ID:** 017@n
```

Comment:
Please update the test case id for other test cases 
please remove from the test suit

Reply:
updated

Reply:
Please update the other test case also according to the interface file i.e. what need return for sink and source device.